docs/user-guide/configuration/settings.md

```diff
@@ -29,4 +31,61 @@ jobs:
                 statuses: [SUCCESS, FAILURE]
 ```
 
-The settings email configuration can be set in `shared`, to apply to all jobs, or in an individual job. A job `email` configuration will completely override the `shared` setting.
\ No newline at end of file
+The settings email configuration can be set in `shared`, to apply to all jobs, or in an individual job. A job `email` configuration will completely override the `shared` setting.
+
+## Slack
+To enable Slack notifications to be sent as a result of build events, invite the Slack user `screwdriver-bot` to your channel(s) and use the Slack setting in your Screwdriver yaml.
+You can configure a list of one or more Slack channels to notify. You can also configure when to send a slack notification, e.g. when the build status is `SUCCESS` and/or `FAILURE`. If you don't configure the build status, it'll default to sending notifications on `FAILURE` only.
+
+### Examples
+
+This simple Slack setting will only send Slack notifications to `mychannel` on build failures:
+
+```
+shared:
+    template: example/mytemplate@stable
+
+jobs:
+    main:
+        settings:
+            slack: 'mychannel'
+```
+
+This Slack setting will send Slack notifications to `mychannel` and `my-other-channel` on build failures:
+
+```
+shared:
+    template: example/mytemplate@stable
+
+jobs:
+    main:
+        settings:
+            slack:
+                channels: ['mychannel', 'my-other-channel']
+```
+
+This Slack setting will send Slack notifications to `mychannel` and `my-other-channel` on all build statuses:
```

Comment:
The status thing is same behavior for email and slack, maybe we can say this part in the common section.